docs/modules/releasenotes/partials/release-notes-26.1.adoc

```diff
@@ -42,3 +42,23 @@ These methods are:
 * `cssClassAsArray`: splits a `cssClass`-string into an array of css classes
 
 In addition, similar methods have been added to the `Cell` class to easier query and modify the `cssClass` property: `addCssClass`, `removeCssClass`, `toggleCssClass`, `hasCssClass`.
+
+== Session API changes
```

Comment:
- move chapter to migration-guide-26.1.adoc file
- leave just a sentence about moving towards a stateless server session here as part of release notes